packages/block-library/src/embed/block.json

```
@@ -0,0 +1,37 @@
+{
+	"name": "core/embed",
+	"category": "embed",
+	"attributes": {
+		"url": {
+			"type": "string"
+		},
+		"caption": {
+			"type": "string",
+			"source": "html",
+			"selector": "figcaption"
+		},
+		"type": {
+			"type": "string"
+		},
+		"providerNameSlug": {
+			"type": "string"
+		},
+		"allowResponsive": {
+			"type": "boolean",
+			"default": true
+		},
+		"responsive": {
+			"type": "boolean",
+			"default": true
```

Per b0755cc2a3d0cbb3ef27b6c45a80b56ab068e345, don't forget:

```suggestion
			"default": false
```